virtualhome/grails-app/views/lostPassword/emailed.gsp

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html>
+  <head>
+    <meta name="layout" content="myaccount" />
+  </head>
+  <body>
+
+    <div class="hero-unit">
+      <h2>If you have an account here, an email has been sent to you.</h2>
+
+      <g:render template="/templates/flash" plugin="aafApplicationBase"/>
+
+      <p>Please follow the instructions in the email.</p>
+
+      <h3>Didn't receive the email?</p>
+
+      <g:form action="emailed">
+        <g:hiddenField name="login" value="${login}"/>
+        <button type="submit" class="btn btn-large">Send another email.</button>
+        <g:link controller="lostpassword"></g:link>
+      </g:form>
+
+      <h3><strong>NOTE: </strong>Only the most recently sent email will contain a vaild URL</h3>
```

Comment:
Sorry, a typo in one word ("vaild") ... and suggesting a slight rewording, plus using the style from getbootstrap used elsewhere for warning notices:
```suggestion
      <div class="alert alert-block alert-warning"><h4>Note</h4>Only the URL from the most recent email will be valid.</div>
```

Attaching the rendered result (played with browser tools in F
![Screenshot 2024-11-07 at 08 57 04](https://github.com/user-attachments/assets/838bd206-8b22-4462-817b-2def6a9958a8)
irefox)